site/docs/plugins/ratelimiter.md

```diff
@@ -24,9 +27,14 @@ This plugin exposes 5 customisable options:
 
 ### About `storageClient`
 
-The `MEMORY_STORE` or the in-memory tracking is suitable for most bots, however if you implement clustering for your bot you will not be able to use the in-memory storage effectively. That's why the Redis option is provided as well. You can pass a Redis client from [ioredis](https://github.com/luin/ioredis) or [redis](https://deno.land/x/redis) in case you use deno. In reality any Redis driver that implements the `incr` and `pexpire` methods should work just fine. rateLimiter is driver agnostic.
+The `MEMORY_STORE` or the in-memory tracking is suitable for most bots, however if you implement clustering for your bot you will not be able to use the in-memory storage effectively.
+That's why the Redis option is provided as well.
+You can pass a Redis client from [ioredis](https://github.com/luin/ioredis) or [redis](https://deno.land/x/redis) in case you use deno.
```

Comment:
```suggestion
You can pass a Redis client from [ioredis](https://github.com/luin/ioredis) or [redis](https://deno.land/x/redis) in case you use Deno.
```